### PR TITLE
fix: studio breaking when trying to generate code/document

### PR DESCRIPTION
**Pull request by AceTheCreator**
This PR fixes the following issue. #803 

cc @fmvilas @Souvikns

### CODE REVIEW

File: apps/studio/src/components/Modals/Generator/TemplateParameters.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const TemplateParametersSans: ForwardRefRenderFunction<TemplateParameters
     const servers = document?.servers();
     const availableServers: string[] = [];
     Object.entries(servers || {}).forEach(([serverName, server]) => {
-      if (supportedProtocols.includes(server.protocol())) availableServers.push(serverName);
+      if (server.protocol && supportedProtocols.includes(server.protocol())) availableServers.push(serverName);
```
Comment (fmvilas) [inline apps/studio/src/components/Modals/Generator/TemplateParameters.tsx:109]: How can it be that `protocol` is not a function? What is it instead? 🤔

**Comment by fmvilas** [comment]
Also, if you plan to release a new version, please add a changeset: https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

File: apps/studio/src/components/Modals/Generator/TemplateParameters.tsx (hunk 1 above)
Comment (AceTheCreator) [reply apps/studio/src/components/Modals/Generator/TemplateParameters.tsx:109]: So, based on what I noticed during the iteration, some index of server in `server.protocol` is returning `undefined`

**Comment by AceTheCreator** [comment]
> Also, if you plan to release a new version, please add a changeset: https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

Why should it be a new version if I'm just fixing a tiny bug 🤔

**Comment by KhudaDad414** [comment]
@AceTheCreator it is a bug fix so it should release a patch version.

**Review by KhudaDad414 — APPROVED** [review]
LGTM!

**Comment by AceTheCreator** [comment]
@fmvilas, can you approve this?

**Pull request merged**